core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -1600,6 +1600,42 @@ secret_configs:
     - non_app_ci
     name: cluster-secrets-azure-2
     namespace: ci
+- from:
+    osServicePrincipal.json:
+      field: osServicePrincipal.json
+      item: os4-installer.openshift-ci2.azure
```

Comment:
So what is actually the MAG part of the profile? Is it supposed to use the same auth credentials like the `cluster-secrets-azure-2` one? If so then you don't need to add it as a copy, just as an item to the `to:` list of that secret.